### DISCUSSION TITLE
2.14.1

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#749] [Releases]
- Hide search, expand and sort icons until there is at least 1 card
- Various theming fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/CatimaLoyalty/Android/releases/tag/v2.14.1'>2.14.1</a>.</em>

**Comment by Altonss** [comment]
I don't know if it's intentional, but the new Material You redesign changed the accent color of catima to a more light pink in the dark theme, and I find it not so beautiful. Could it be possible to reconsider the colour of the dark theme, because as of know it's really light :/ Especially the top colour (in the notification bar, or the accent colour for some text or buttons.
But appart from this, I really like the new design :smile:  Thanks for all the work!

Here some screenshots:
Before
![image](https://user-images.githubusercontent.com/66519591/149975844-99817292-ec1d-43f5-b0f2-773c10db1c8f.png)

After
![image](https://user-images.githubusercontent.com/66519591/149975673-9ec98712-3d09-4a53-86a9-3aff13064902.png)
I find the old look much more sober, calm, appeasing and elegant.

**Reply by TheLastProject** [reply]
Yeah I'm not a big fan either, but that is what https://material-foundation.github.io/material-theme-builder/ generated when I gave it "Catima Red" (#A83536). I am willing to accept a pull request that tweaks this a bit, preferably in a way of finding a setting that generates a nicer dark theme on the material theme builder.

**Reply by Altonss** [reply]
Maybe for the next big updates, it would be cool to discuss the evolution beforhand so we can discuss about the issues (for example UI/UX) :smile:. Because this change came really fast without having had the opportunity to discuss the new colour changes.
It' s not a criticism at all, but just an improvement remark so that the evolution process of Catima could be even smoother for the developers as well as for the numerous users :)
